### ISSUE TITLE
actions.ini bringup edit

### ISSUE THREAD

**Issue by dougransom** [dictation-toolbox/unimacro#3]
The `[bringup edit]` section defaults to programs most users will not have on their system.  so on a clean install they have little chance of getting unimacro working without help.

Propose to default to "notepad" in the default actions.ini.

**Comment by quintijn** [comment]
This point ask for a lot of rethinking!

**Comment by quintijn** [comment]
Default to Notepad is always a good idea

**Comment by dougransom** [comment]
That is better.

Question:  Wouldn't it be easier/better to rely on windows defaults?   The user already has their preference set in windows, and now they are fighting that behavior with an ini file.

**Comment by quintijn** [comment]
Yes, I will go over this when most other things work in Unimacro...
